multi-node/aws/artifacts/scripts/install-controller.sh

```diff
@@ -37,16 +34,16 @@ ENV_FILE=/run/coreos-kubernetes/options.env
 
 function template {
 	# use a heredoc so the quoting & whitespace in the
-	# downloaded artifact is preserved, but env variables
+	# artifact is preserved, but env variables
 	# can still be evaluated
 	eval "cat <<EOF
-$(curl --silent -L "${ARTIFACT_URL}/$1")
+$(cat $1)
```

Comment:
I don't know a good alternative to this off the top of my head, but eval'ing a heredoc that is the subshell output of cat'ing a file ... seems like there must be a better way.

Quick googling brings up envsubst, but looks like that my not be available in coreOS.

I won't block on this, but maybe just do a quick search to see if there's a better way of doing this...